packages/builder/src/stores/portal/apps.ts

```diff
@@ -1,19 +1,39 @@
 import { derived } from "svelte/store"
+// @ts-ignore
```

Comment:
Needed because the tsconfig refuses to properly map "constants" to "src/constants/index.js" even if specifying an explicit path. It instead prefers to resolve this to some constants module from node_modules and throws an error saying AppStatus does not exist.

Reply:
On this one - I had a look into this, it can be aliased, but it appears we just need to update the constants alias to `@constants` - I'm not sure where the overlap it coming from exactly, but splitting it feels like a better option to disabling Typescript completely.

Reply:
After a discussion - going to approve this PR with the ignore, but we're going to update all aliases to follow the `@` pattern often seen with aliases - this avoids overlap with builtin Node modules which cannot be overridden.

Reply:
I found this issues to... 🤷🏽